backend/oasst_backend/api/v1/leaderboards.py

```diff
@@ -1,26 +1,50 @@
-from fastapi import APIRouter, Depends
+from typing import Optional
+
+from fastapi import APIRouter, Depends, Query
 from oasst_backend.api import deps
 from oasst_backend.models import ApiClient
-from oasst_backend.user_repository import UserRepository
+from oasst_backend.user_stats_repository import UserStatsRepository, UserStatsTimeFrame
 from oasst_shared.schemas.protocol import LeaderboardStats
 from sqlmodel import Session
 
 router = APIRouter()
 
 
-@router.get("/create/assistant")
-def get_assistant_leaderboard(
+@router.get("/day")
```

Comment:
all these routes could probably be unified to save some code

Reply:
ok